evaluation/README.md

```diff
@@ -0,0 +1,96 @@
+# Evaluation Metrics
```

Comment:
Would add instructions on how to run in readme

Reply:
I'm going to hold off on doing this until we make a decision if we are going to publicize TruLens or not. For now I'm the only one running these and I know how to do so.